### PR TITLE
feat(settings): add github webhook secret

### PR DESCRIPTION
**Pull request by jan-zabloudil**

### CODE REVIEW

File: repository/model/settings.go [hunk 1]
```diff
@@ -75,17 +79,20 @@ func ToSvcSettings(sv []SettingsValue) (svcmodel.Settings, error) {
 				return svcmodel.Settings{}, err
 			}
 		case keySlack:
-			if err := json.Unmarshal(settingsValue.Value, &s.Slack); err != nil {
+			if err := json.Unmarshal(settingsValue.Value, &slackSettings); err != nil {
 				return svcmodel.Settings{}, err
 			}
 		case keyGithub:
-			if err := json.Unmarshal(settingsValue.Value, &s.Github); err != nil {
+			if err := json.Unmarshal(settingsValue.Value, &githubSettings); err != nil {
```
Comment (jan-zabloudil) [inline repository/model/settings.go:86]: I need to first unmarshal it into a struct from the repository model because this model contains the JSON tags.

**Review by bolshakoff — APPROVED** [review]

**Review by bolshakoff — COMMENTED** [review]
Un-approving now, because in a neighbor PR the discussion goes in a direction where we might not need hooks at all.

**Comment by bolshakoff** [comment]
#109; [comment](https://github.com/jan-zabloudil/release-manager/pull/109#discussion_r1616182848)

**Comment by jan-zabloudil** [comment]
Hooks will be used - https://github.com/jan-zabloudil/release-manager/pull/109#issuecomment-2133989882

**Pull request merged**